mtest/libmscore/all_elements/tst_tree_model.cpp

```diff
@@ -23,28 +23,31 @@ using namespace Ms;
 ///   Ensures that the tree model is consistent. Starting
 ///   from Score each element in the tree should have the
 ///   correct parent element, the one whose children list
-///   it appears in.
+///   it appears in. 
 //---------------------------------------------------------
 
 class TestTreeModel : public QObject, public MTest
 {
     Q_OBJECT
 
-    MasterScore* score;
-    void beam(const char* path);
+    MasterScore * score;
```

Comment:
Unnecessary space.

Reply:
Oops, I thought I had fixed it. Maybe the old version came back because of some merge conflict while rebasing. Anyways the test works without this line so I'll just remove it.